app/views/nsm/steps/disbursements/edit.html.erb

```diff
@@ -27,14 +26,14 @@
                   <% if OtherDisbursementTypes.values.include?(OtherDisbursementTypes.new(disbursement.other_type)) %>
                     <%= t("helpers.other_disbursement_type.#{disbursement.other_type}") %>
                   <% else %>
-                    <%= check_missing(disbursement.other_type)  %>
+                    <%= check_missing(disbursement.other_type) %>
                   <% end %>
                 <% else %>
                   <%= check_missing(disbursement.disbursement_type) { t("helpers.label.nsm_steps_disbursement_type_form.disbursement_type_options.#{disbursement.disbursement_type}") } %>
                 <% end %>
               </td>
               <td class="govuk-table__cell"><%= check_missing(disbursement.total_cost) { NumberTo.pounds(disbursement.total_cost) } %></td>
-              <td class="govuk-table__cell"><%= check_missing(disbursement.details)  %></td>
+              <td class="govuk-table__cell"><%= check_missing(disbursement.disbursement_date.present? && (disbursement.prior_authority.present? || !disbursement.auth_required?) && disbursement.details) %></td>
```

Comment:
your going tot hate me... should this be extracted to a method?

Reply:
Ugh... yes it should....